__tests__/__theia__/extension.theia.test.ts

```diff
@@ -9,51 +9,266 @@
 *                                                                                 *
 */
 
-import { Builder, By, Key, until } from "selenium-webdriver";
-// tslint:disable-next-line:no-submodule-imports
-import * as firefox from "selenium-webdriver/firefox";
 import * as chai from "chai";
 import * as chaiAsPromised from "chai-as-promised";
+import * as driverFF from "../../src/theia/extension.theiaFF";
+import * as driverChrome from "../../src/theia/extension.theiaChrome";
+import { sleep } from "@zowe/cli";
 
 const TIMEOUT = 45000;
 const SLEEPTIME = 10000;
-const WAITTIME = 30000;
+const SleepForTwoSec = 2000;
```

Comment:
Why was this changed? Kinda nice to have all the constants use the same capitalization ^^' Maybe we could call it SHORTSLEEPTIME or something? idk

Reply:
I have Changed variable name.